default.vcl

```diff
@@ -202,4 +202,16 @@ sub vcl_deliver {
     } else {
         set resp.http.X-Cache = "MISS";
     }
+
+    # CORS response for smartlogic widget
+    if (req.http.Origin == "https://cloud.smartlogic.com") {
```

Comment:
This will allow smartlogic to access cross-origin any of our endpoints in delivery - should we restrict it to the endpoints it needs?

Reply:
Ah yes if possible, it would be better. 
I think we can restrict it if we know all names of backend like this.

```
sub vcl_backend_response {
  ...
  set beresp.http.X-Backend-Name = beresp.backend.name;
}

sub vcl_deliver {
  ...
  if (req.http.Origin == "https://cloud.smartlogic.com" && resp.http.X-Backend-Name ~ (public-content-by-concept-api|concept-search-api|public-concordances-api)) {
    # set cors headers
  }
}
```

But the problem is that there is no `public-concordances-api` backend in this VCL. Can I detect concordance api backend on this VCL? Please give some advice. @peteclark-ft